JSDemos/Demos/Resizable/Overview/description.md

```diff
@@ -1 +1,21 @@
-The Resizable is a UI component that enables its content to be resizable in the UI.
\ No newline at end of file
+The Resizable enables you to resize its content. In this demo you can drag [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) to resize the DataGrid.
+
+Configure the following properties to specify the boundaries to which you can drag the handles:
+
+- [minWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minWidth)    
+The lower width boundary.
+
+- [maxWidth](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxWidth)    
+The upper width boundary.
+
+- [minHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#minHeight)    
+The lower height boundary.
+
+- [maxHeight](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#maxHeight)    
+The upper height boundary.
+
+You can specify four types of handles: *top*, *bottom*, *left* and *right*. Separate handles by a space in the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) property to specify which handles to display in the UI. Combine nearest handles (for example, "bottom left") to show corner handles.
+
+The [keepAspectRatio](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#keepAspectRatio) property specifies whether to resize the Resizable's content proportionally with corner handles. If you need to disable propotional resize, set this property to **false**.
```

Comment:
(Concise Text) - in the first sentence I want to rephrase the last part: "resize the Resizable's content proportionally with corner handles". 
(Repetition) - then I can avoid the repetition of the word "proportional" in the second sentence. ("Free-form" - is one of the picture resize modes in iOS.)

```suggestion
The [keepAspectRatio](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#keepAspectRatio) property specifies whether a corner handle resizes content proportionally. If you need to allow free-form resize operations, set this property to **false**.
```